src/mlpack/core/data/dataset_info.hpp

```diff
@@ -2,9 +2,9 @@
  * @file dataset_info.hpp
  * @author Ryan Curtin
```

Comment:
If you're going to rename `DatasetInfo` to `DatasetMapper`, you should probably rename this file to `dataset_mapper.hpp` and `dataset_info_impl.hpp` to `dataset_mapper_impl.hpp`.  Also you should probably add your name to the list of authors. :)